package.json

```diff
@@ -56,7 +56,7 @@
     "react-json-view": "^1.21.3",
     "react-modal": "^3.16.1",
     "react-modal-promise": "^1.0.2",
-    "react-router-dom": "^6.27.0",
+    "react-router-dom": "^7.0.1",
```

Comment:
This is not what the PR description says.

Reply:
Bad bot

Reply:
Pushed the change manually